content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst

```diff
@@ -0,0 +1,153 @@
+======================
+Two-step manufacturing
+======================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
+two-step manufacturing, Odoo creates a manufacturing order and a pick components transfer, but does
+not generate a transfer for the movement of finished products into stock. Inventory counts still
+update based on the number of products manufactured, but the act of transferring them to and from
+inventory is not tracked.
```

Comment:
```suggestion
not generate a transfer for the movement of finished products into stock. Inventory counts of
manufactured goods still update based on the number of products manufactured, but the act of
transferring them between the *Manufacturing* and *Inventory* apps is not tracked.
```

The sentence works as is, since I understood it after re-reading it a few more times, but I was thinking that there could be a bit more clarification about why the products are getting moved in the first place. When I first read this sentence, I was confused about *why* the products are getting moved between inventory and manufactured.

So, I suggested something that made a little bit more sense to me, but I think you can clarify best!